spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "2.1"
+services:
+  spark:
+    image: feathub-spark:latest
```

Comment:
Instead of using a custom image here, would it be simpler to use the  apache/spark 3.3.1 image to start master/worker?

The container can load data to the /tmp/data directory. And we can ask users to copy the data to the /tmp/data directory so that both driver program and spark worker can access data at /tmp/data.

I verified that it works by starting a container using the command `docker run -p 7077:7077 -p 8080:8080 --user root -it -v `pwd`/data:/tmp/data apache/spark:3.3.1 bash`.

This approach allows us to simplify the PR as users do not need to build the docker image to run this example. And the Spark example's setup would also be more consistent with that of Flink examples.